packages/plugin-web-attribution-browser/src/helpers.ts

```diff
@@ -17,6 +17,15 @@ const domainWithoutSubdomain = (domain: string) => {
   return parts.slice(parts.length - 2, parts.length).join('.');
 };
 
+const isDirectTraffic = (current: Campaign) => {
+  const { referrer, referring_domain, ...currentCampaign } = current;
+  Object.keys(currentCampaign).forEach(
+    (key) => (currentCampaign[key] === undefined || currentCampaign[key] === '') && delete currentCampaign[key],
+  );
```

Comment:
do we really need to mutate this? i prefer that we do not mutate. if we're using this for the purposes of a conditional, we can just use `filter` or `every`

Reply:
update to `  const currentCampaignWithValue = Object.keys(currentCampaign).filter(key => (currentCampaign[key] === undefined || currentCampaign[key] === ''))`